build.xml

```diff
@@ -21,11 +21,11 @@
     <property name="svm.build"   location="${svm.dir}/mxbuild/dists" />
     <property name="truffle.build" location="${truffle.dir}/mxbuild/dists" />
     <property name="somns-deps.version" value="0.3.6" />
-    <property name="checkstyle.version" value="8.11" />
+    <property name="checkstyle_checkse.version" value="8.11" />
```

Comment:
Why this change?

Reply:
I think this was generated by mx.

Reply:
No, please revert. mx doesn't know anything about our build system.